.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,141 @@
+
+### I have an enhancement 
+- Submit issue using the enhancement request template
+  - please read "[the good: issue](#issue)"
+- Contact kirby-guild to get it tech refined 
+- If you would like to implement issue: 
+  -please read: 
+    - "[Setup environment and tooling for contribution](#setup-environment-and-tooling-for-contribution)" to get properly setup 
+    - "[Creating pull-requests](#creating-pull-requests)" to get informed on how to most easily get your changes merged
+  - assign yourself to the issue to signal to others you are working on it
+  - if you at any point have questions while solving the issue please see: [I have a question related to Kirby](#i-have-a-question-related-to-kirby)
+
+### I would like to report a bug 
+- Submit issue using the bug template  
+  - please read "[the good: issue](#issue)"
+- Contact kirby-guild to get it verified 
+  - we verify because it might be a feature not a bug 🐛
+- If you would like to implement issue: 
+  -please read: 
+    - "[Setup environment and tooling for contribution](#setup-environment-and-tooling-for-contribution)" to get properly setup 
+    - "[Creating pull-requests](#creating-pull-requests)" to get informed on how to most easily get your changes merged
+  - assign yourself to the issue to signal to others you are working on it
+  - if you at any point have questions while solving the issue please see: [I have a question related to Kirby](#i-have-a-question-related-to-kirby)
+
+## Setup environment and tooling for contribution
+<!-- The following section is responsible for teaching people how to get setup with 
+the repo such that they can get coding -->
+
+### Clone and install repository 
+- how to clone and get going with the kirby repo
+
+### Node 
+- Which version should people use, and perhaps a recommendation on which tool people can use to manage versions. 
+
+### Tslint 
+- People should make sure to have it enabled to avoid unnecessary comments during code review
+
+### Prettier
+- It is a good idea to use format-on-save
+
+## Creating pull-requests 
+<!-- This section is responsible for guiding contributors on how they should submit pull-requests in a way that ensures the most swift process and teach them how to catch low-hanging issues there might come up in code-review. -->
+
+### Before you submit  
+- Make sure you have completed the steps in: [Setup tooling for contribution](#setup-tooling-for-contribution)
+- Make sure you have written tests according to the guidelines described in [The good: test](#test)
+- Do a self-review <!-- We could link to self-review in the templates and have people check it off? -->
+
+### After you submit 
+- Post in Kirby-guild 
+- Code reviews are meant to align your code with existing code 
+  - While your implementation might be perfectly valid, we are aiming for uniformity across the codebase. 
+- Don't commit and exit - make sure to stay responsive and come back to answer comments on your code review
+
+## The good:
```

Comment:
Maybe add something like 
### Commit
 - Focus on one thing per commit
 - Keep changes as small as possible
 - Only commit related functionality for each commit
 - Commit often
 - Strive for each commit to compile/build and pass tests

Reply:
And maybe also add that the CI pipeline handles combining each commit to a merge commit containing all commit messages

Reply:
e15cb28 😄